mongo/tests/compose/certs/server.pem

```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
+MIIBEzCBugIJALu3/pYL70S9MAoGCCqGSM49BAMCMBIxEDAOBgNVBAMMB2RhdGFk
+b2cwHhcNMjIwNzE5MTIxMjA1WhcNMjUwNDE0MTIxMjA1WjASMRAwDgYDVQQDDAdk
+YXRhZG9nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEOr/mu0dRG1KT6j+LYlwo
+KOK7hZoVzF7AmV83HOU4jFHMTcXCKIi2dYu/lsqEJbQJVHiRLTcdqkmM6BaehCKk
+bTAKBggqhkjOPQQDAgNIADBFAiBekcvpyzqHklrS9Zrv3z8zquJcFoQEP5JMfXok
+BnuNpQIhAMJ6tkxsNX0V2/nEkaVZgz8hHA6oKMyRS1o+A2Lcxyha
+-----END CERTIFICATE-----
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIB/ztjw107tUms4kI7Ig64WvAMqRpAJuBzSkukI7MfpJoAoGCCqGSM49
+AwEHoUQDQgAEOr/mu0dRG1KT6j+LYlwoKOK7hZoVzF7AmV83HOU4jFHMTcXCKIi2
+dYu/lsqEJbQJVHiRLTcdqkmM6BaehCKkbQ==
+-----END EC PRIVATE KEY-----
```

Comment:
Do you know when they will expire?

Reply:
I have updated certificates to expire in 2122